### PR TITLE
HARP-8309 Get LTS at world pos

### PR DESCRIPTION
**Pull request by nzjony**
Signed-off-by: Jonathan Stichbury <2533428+nzjony@users.noreply.github.com>

### CODE REVIEW

File: @here/harp-geoutils/lib/math/MathUtils.ts [hunk 1]
```diff
@@ -63,6 +63,22 @@ export namespace MathUtils {
...
+    /**
+     * Copies the vector across.
+     *
+     * @param from The vector to copy from.
+     * @param to The resulting [Vector3Like] instance, with the contents copied from from
+     */
+    export function copyVector3<Vector extends Vector3Like>(from: Vector3Like, to: Vector): Vector {
+        if (from === undefined || to === undefined) {
+            throw new Error("from or to vector is/are empty");
+        }
```
Comment (robertoraggi) [inline @here/harp-geoutils/lib/math/MathUtils.ts:75]: I don't think this checks are needed.

File: @here/harp-geoutils/lib/projection/MercatorProjection.ts [hunk 2]
```diff
@@ -328,6 +328,15 @@ class WebMercatorProjection extends MercatorProjection {
         MathUtils.newVector3(0, 0, -1, result.zAxis);
         return result;
     }
+
+    /** @override */
+    localTangentSpaceFromWorld(worldPos: Vector3Like, result: TransformLike): TransformLike {
```
Comment (robertoraggi) [inline @here/harp-geoutils/lib/projection/MercatorProjection.ts:333]: `localTangentSpaceFromWorld` sounds strange to me, how about `localTangetSpaceAtPosition` ?
Alternatively you could overload the signature of localTangentSpace, e.g.
```
localTangentSpace(position: GeoCoordinatesLike | Vector3Like, result: TransformLike) {
    if (!isVector3Like(position)) {
        position = this.projectPoint(position);
   }
   // ...
```

**Comment by nzjony** [comment]
@harpgl-bot retest this please

File: @here/harp-geoutils/lib/projection/MercatorProjection.ts (hunk 2 above)
Comment (ninok) [reply @here/harp-geoutils/lib/projection/MercatorProjection.ts]: I would also prefer to just add an overload for the existing method

Comment (ninok) [reply @here/harp-geoutils/lib/projection/MercatorProjection.ts]: just saw that you did this already. Should look at the code before commenting next time ;-)

**Pull request merged**